content/docs/identity-providers/azure.mdx

```diff
@@ -1,156 +1,177 @@
 ---
-title: Azure Active Directory
-sidebar_label: Azure AD
+title: Entra ID (Azure Active Directory)
+sidebar_label: Entra ID
+description: Learn how to configure Microsoft Entra ID (formally Azure AD) as an identity provider that works with Pomerium Core and Enterprise.
 lang: en-US
-keywords: [azure, active directory, ad, microsoft, identity provider, idp]pagination_prev: null
+keywords:
+  [
+    azure,
+    active directory,
+    ad,
+    entra,
+    entra id,
+    microsoft,
+    identity provider,
+    idp,
+  ]
+pagination_prev: null
 pagination_next: null
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This page will help you configure Pomerium to use your Azure Active Directory as an IdP. It assumes you have already [installed Pomerium](/docs/deploy/)
+# Microsoft Entra ID (Azure Active Directory)
```

Comment:
I think we can remove this explicit heading line and use the default title instead?

Reply:
So this is more of an SEO thing. The `title` front matter is treated as the title tag (what you'd see in the browser tab). If you don't include an H1 like I have, Docusaurus will use the `title` front matter value as the H1. Specifically for this guide, including both is redundant, but sometimes the H1 should differ from the title (like the code-server guide). I do this out habit to make sure future pages are treated the same.

Reply:
Do you want to update this heading to match the current title then?